### PR TITLE
Add `ImageDraw.regular_polygon` to release notes

### PR DESCRIPTION
**Pull request by comhar**
Add `ImageDraw.regular_polygon` to release notes. 

[Reference](https://github.com/python-pillow/Pillow/pull/4846#issuecomment-687366543)

### CODE REVIEW

**Review by hugovk — APPROVED** [review]
Thanks!

File: docs/releasenotes/8.0.0.rst [hunk 1]
```diff
@@ -78,6 +78,14 @@ high ends.
 
 Now, it can also be a tuple ``(low, high)``.
 
+ImageDraw.regular_polygon
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A new method ``regular_polygon``, draws a regular polygon of ``n_sides``, inscribed in a ``bounding_circle``.
```
Comment (nulano) [inline docs/releasenotes/8.0.0.rst:84]:
```suggestion
A new method :py:meth:`.ImageDraw.regular_polygon`, draws a regular polygon of ``n_sides``, inscribed in a ``bounding_circle``.
```

Let's link to the method documentation.

File: docs/releasenotes/8.0.0.rst [hunk 2]
```diff
@@ -78,6 +78,14 @@ high ends.
 
 Now, it can also be a tuple ``(low, high)``.
 
+ImageDraw.regular_polygon
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A new method ``regular_polygon``, draws a regular polygon of ``n_sides``, inscribed in a ``bounding_circle``.
+
+For example ``ImageDraw.regular_polygon(((100, 100), 50), 5)``
```
Comment (radarhere) [inline docs/releasenotes/8.0.0.rst:86]:
```suggestion
For example ``draw.regular_polygon(((100, 100), 50), 5)``
```

**Comment by hugovk** [comment]
Thank you!

**Pull request merged**